site/src/component/GradeDist/Pie.tsx

```diff
@@ -102,7 +104,7 @@ export default class Pie extends React.Component<PieProps> {
         id: 'A',
         label: 'A',
         value: gradeACount,
-        color: '#60A3D1',
+        color: '#54B058',
```

Comment:
consider using something like `colors[0]` for this so we can make these use the same source for colors